openy_memberships.install

```diff
@@ -114,4 +114,14 @@ function openy_memberships_update_8004() {
       'field.storage.profile.field_age_group.yml',
       'field.field.profile.family_members.field_age_group.yml',
     ]);
-}
\ No newline at end of file
+}
+
+/**
+ * Update config for front app after removing DiscountFinder step.
+ */
+function openy_memberships_update_8005() {
```

Comment:
this won't work for installation from scratch. We need to update initial config as well

Reply:
Proof https://sandbox-carnation-std-membership-framework-d9.openy.org/membership-builder#/ @szherebchuk

Reply:
Please, review https://github.com/ymcatwincities/openy_memberships/pull/97 @szherebchuk @anpolimus